2020-02-10-OpenFF-Group1-Torsions-3/03_create_torsiondrive_dataset.py

```diff
@@ -0,0 +1,132 @@
+import os
+import json
+import tarfile
+import tqdm
+
+import qcportal as ptl
+import qcelemental as qcel
+from qcelemental.models import Molecule
+
+collection_name = "OpenFF Group1 Torsions 2"
```

Comment:
Please do make sure to update the collection name. If I forget to do it, this could be bad!

Reply:
Oh no, my bad! Thanks for fixing it!